mainnet1/mainnet1.toml

```diff
@@ -44,7 +44,7 @@ Value = "mainnet"
       # "https://vega.greenfield.xyz",
       # !! not passing standard validations
       # "https://vega.xprv.io/datanode",
-      # "https://vega.mainnet.stakingcabin.com",
+      "https://vega.mainnet.stakingcabin.com",
```

Comment:
This one doesn't work for me, but `https://vega.mainnet.stakingcabin.com:3008` works.

Reply:
changed